rcaracaus/pledge-vegan-node
# 29-es-lint (2)[1]

Steps to test:

1. Go to localhost:3000 and check that nothing is wrong and every page and sub directory loads correctly.
2. Run eslint on the lib folder and make sure there are no errors.
